sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -541,6 +541,15 @@ object OverwritePartitionsDynamic {
   }
 }
 
+case class DeleteFromTable(
+    child: LogicalPlan,
+    condition: Filter)
+    extends Command {
```

Comment:
nit: we don't need to put this in a new line.

Reply:
Fixed.